tests/utils.js

```diff
@@ -1017,4 +1017,5 @@ module.exports = {
   },
 
   runAndLogApiStartupMessage: runAndLogApiStartupMessage,
+  findDistrictHospitalFromPlaces: (places) => places.find((place) => place.type === 'district_hospital')
```

Comment:
Nice. Please also use this new function to replace the same code over in [person-under-area.wdio-spec.js](https://github.com/medic/cht-core/blob/a6903096ebffa4cf7cbe39ea481e804bb0707088/tests/e2e/contacts/person-under-area.wdio-spec.js#L9-L10).

Reply:
Done